app/views/components/cases_cards.scala.html

```diff
@@ -28,10 +28,10 @@
 @(countCases: Map[String, Int], referredCasesAssignedToMe: Int, completedCasesAssignedToMe: Int)(implicit request: AuthenticatedRequest[_], messages: Messages, appConfig: AppConfig)
 
 <div id="dashboard-view" class="centered-content">
-
-    <span class="caption-xl mt-40">@Role.format(request.operator.role)</span>
-    <h1 class="heading-xlarge mt-0 mb-0">@request.operator.name.getOrElse("")</h1>
-
+    <div class="operation-dashboard-header column-two-thirds">
+        <span class="caption-xl mt-40">@request.operator.name.getOrElse("")</span>
```

Comment:
Is there a reason why the role has been removed?  This was added as per requirement for the operator-dashboard to display the role of the logged in user?

Reply:
@emilia-trendafilova the requirement is replace role to 'Case dashboard'